packages/orbit-components/src/InputSelect/index.tsx

```diff
@@ -212,19 +215,25 @@ const InputSelect = React.forwardRef<HTMLInputElement, Props>(
         autoFocus={!isLargeMobile}
         role="combobox"
         value={inputValue}
-        onKeyDown={ev => {
-          if (onKeyDown) onKeyDown(ev);
-          handleDropdownKey(ev);
-        }}
+        onKeyDown={
```

Comment:
I was thinking for a while if it's not the same flow as we have with `onFocus`. If the `onFocus` prop is passed to `InputSelect` component, then we pass it to `InputField`. But that's applicable only when `readOnly` is false, it doesn't make sense to have `onKeyDown` available in `readOnly` mode. Am I correct?

@DSil